### PR TITLE
Feature/my cases

### PR DESCRIPTION
**Pull request by WeiWu2**
### **Changes:**
- My Cases table will now auto refresh on upload success and cancel
- added a refresh button on My Cases table to manually refresh the table

### CODE REVIEW

File: src/components/pages/MyCases/MyCases.js [hunk 1]
```diff
@@ -9,40 +9,19 @@ import Box from '@material-ui/core/Box';
...
-          })
-        );
-      })
-      .catch(err => {
-        console.log(err);
-      });
-  }, [user.authState.idToken.idToken]);
+    getPendingCases();
+    // eslint-disable-next-line
+  }, []);
```
Comment (katieolson84) [inline src/components/pages/MyCases/MyCases.js:24]: Nice work moving the axios call to clean up this component.

File: src/components/pages/Home/RenderHomePage.js [hunk 2]
```diff
@@ -157,7 +157,31 @@ function RenderHomePage(props) {
     // savedCases.length,
     // savedJudges.length,
   ]);
-
+  const getPendingCases = () => {
```
Comment (katieolson84) [inline src/components/pages/Home/RenderHomePage.js:160]: Note for the future:  we should move these axios calls into a separate folder or implement redux.

File: src/components/pages/MyCases/MyCases.js [hunk 3]
```diff
@@ -67,7 +46,8 @@ export default function MyCases(props) {
       .catch(err => {
         console.log(err);
       });
-  }, [user.authState.idToken.idToken]);
+    // eslint-disable-next-line
```
Comment (katieolson84) [inline src/components/pages/MyCases/MyCases.js:49]: Nice job adding the eslint line to ignore the error!

**Review by katieolson84 — APPROVED** [review]
I am constantly amazed by your coding skills. You do a great job of implementing changes and cleaning up the code as you go. Approved, as no changes need to be made!

File: src/components/pages/MyCases/MyCases.js (hunk 1 above)
Comment (paulstgermain) [reply src/components/pages/MyCases/MyCases.js:24]: Agreed - loving the separation of concerns here!

File: src/components/pages/MyCases/MyCases.js [hunk 4]
```diff
@@ -117,6 +100,8 @@ export default function MyCases(props) {
       headerName: 'Status',
       headerAlign: 'center',
       flex: 1,
+      // current there is a bug with status where having two cases with the same Case Number will change every status
+      // this will be fixed once we figure out what to do with Case Number
```
Comment (paulstgermain) [inline src/components/pages/MyCases/MyCases.js:104]: Good work accounting for potential errors and making notes for the future!

**Review by paulstgermain — APPROVED** [review]
Approved on basis of clean code and adherence to our established requirements for pull requests!

**Pull request merged**